### PR TITLE
Add domain to SSL

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

**Comment by schnie** [comment]
@andriisoldatenko should we .gitignore those pem files?

**Comment by andriisoldatenko** [comment]
nice catch!

File: bin/generate_ssl_keys [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # https://github.com/FiloSottile/mkcert
 mkcert -install
-mkcert app.local.astronomer-development.com "*.local.astronomer-development.com"
-cat app.local.astronomer-development.com+1.pem > /tmp/fullchain.pem
+mkcert app.local.astronomer-development.com local.astronomer-development.com "*.local.astronomer-development.com"
+cat app.local.astronomer-development.com+2.pem > /tmp/fullchain.pem
```
Comment (schnie) [inline bin/generate_ssl_keys]: @andriisoldatenko should we just keep this at +1? Or is there a way to choose a distinct name that `mkcert` generates?

Comment (andriisoldatenko) [reply bin/generate_ssl_keys]: @schnie nice catch done in 0600451006074b87f36f64c2c6b69c221765be3d

**Comment by andriisoldatenko** [comment]
TODO rewrite of URL 
 nginx.ingress.kubernetes.io/configuration-snippet: |
         rewrite /preview https://test.app.example.com$uri permanent;

**Pull request merged**